brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2290 Slider navigation not working when block is not in the viewport

### Description
This change will fix the issue with the slider navigation and pagination not working when the slider is not in the viewport inside the editor. 

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
- Add some blocks/content inside the new blank page/post 
- Add a slider block in a way that it will not be in the viewport/first view on the load of the page. 
- Save changes and reload. 
- Scroll down to the slider. Slider navigations should be visible and working as expected. 

### Checklist:
